### DISCUSSION TITLE
text 和 desc 字段匹配想法

### DISCUSSION THREAD

**Discussion by aoguai** [gkd-kit/gkd#552] [Ideas]
在编写规则过程中我发现对 selector 选择器对 text 和 desc 字段的匹配是大小写敏感的。
例如需要
匹配 `close`，如果大小写不敏感的话可能需要编写成 `text="close"||text="Close"||text="CLOSE"` 
我建议可以添加一个字段来控制对字段的匹配时是否大小写敏感，是否需要忽略字符串中的空格。

这将有利于规则的简化编写。

**Comment by Snoopy1866** [comment]
建议一步到位，直接支持正则表达式

**Comment by lisonge** [comment]
https://github.com/gkd-kit/selector
